src/Translator.js

```diff
@@ -52,6 +52,13 @@ class Translator extends EventEmitter {
 
     let namespaces = options.ns || this.options.defaultNS;
     if (nsSeparator && key.indexOf(nsSeparator) > -1) {
+      const m = key.match(this.interpolator.nestingRegexp);
+      if (m && m.length > 0) {
+        return {
+          key,
+          namespaces,
```

Comment:
Hi @adrai,
I located here an issue !

https://github.com/i18next/i18next/pull/1475/files#diff-d7b95552757249c43b739889204bb12002da36af74ac74506364a713ceecf86eR70

If the namespaces is a string, you made this, but not here.